wg-lts/charter.md

```diff
@@ -0,0 +1,95 @@
+
+With this information, we will investigate changes the Kubernetes project could make to address these needs. A non-exhaustive set of changes to investigate include:
+* lengthening support period for all minor versions
+* lengthening support period for specific minor versions
+* applying security fixes to release branches past current EOL without cutting additional patch releases
+* applying security and "critical" fixes to release branches past current EOL without cutting additional patch releases
+* expanding supported skew
+* improving supported upgrade patterns for clusters at EOL
+
+### In Scope
+
+- Collecting input to better define long term support with regard to Kubernetes releases. This could include:
+  * What is a supported release?
+  * Number of community supported branches.
+  * Duration of community support per supported branch.
+- Working with vendors and other community members to identify ongoing efforts to help end users beyond current community support and identify opportunities for reuse of prior work.
+- Creating and prioritizing a list of areas that require investments to improve long term supportability. This could include:
+  * Upgrade path considerations.
+  * Costs of Kubernetes releases in terms of:
+    * Infrastructure
+    * People
+- Initiate and drive cross-SIG changes related to long term support.
+- Identify ways to that vendors and other community members can better support Kubernetes releases.
+
+### Out of scope
+
+- The lifecycle of projects outside of the Kubernetes org.
+- Designing and executing on changes clearly falling into individual SIG
+  responsibilities. This is a working group, no code implementation is the responsibility of this Working Group.
+- Technical and end-user support:  The WG may make recommendations
+  around support to those responsible for relevant code and responsible
+  for the release engineering operations and automation, but does not
+  own code itself.
+
+## Special Powers
+
+None
+
+## Stakeholders
+
+Any changes identified by this working group will require involvement and investment from the following SIGS:
+
+- SIG Architecture
+  High-level input on requirements.
+- SIG Cluster Lifecycle
+  Input on cluster upgrade mechanics.
+- SIG K8s Infra
+  Input on infrastructure costs related to increased support period(s).
+- SIG Release
+  Input on maintaining older branches and additional releases.
+- SIG Testing
+  Input on testing impacts related to upgrades and longer test periods. 
+  
+Improvements identified will likely involve every SIG, but the list above are identified as the primary stakeholders.
+
+## Deliverables
+
+The artifacts the group is supposed to deliver include:
+- Survey results better describing Kubernetes version use patterns, deployment constraints, and upgrade patterns.
+- Recommend changes Kubernetes SIGS can make that will provide broad benefits in a sustainable/affordable way. This would likely take the form of one or more KEPs.
```

Comment:
nit:
```suggestion
- Recommend changes Kubernetes SIGs can make that will provide broad benefits in a sustainable/affordable way. This would likely take the form of one or more KEPs.
```